### PR TITLE
CONSOLE-2523: Add resource requests to cluster and project overview

### PR DESCRIPTION
**Pull request by rawagner**
Project overview
Screenshot from 2021-01-11 09-27-52

Cluster overview
Screenshot from 2021-01-11 09-35-13

@smarterclayton @spadgett

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx [hunk 1]
```diff
@@ -115,7 +125,12 @@ export const UtilizationItem: React.FC<UtilizationItemProps> = React.memo(
     setTimestamps,
   }) => {
     const { t } = useTranslation();
-    const { data, chartStyle } = mapLimitsRequests(utilization, limit, requested);
+    const { data, chartStyle } = mapLimitsRequests(
+      utilization,
+      limit,
+      requested,
+      trimSecondsXMutator,
```
Comment (rawagner) [inline frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx:132]: @rebeccaalpert this is probably the fix you want for pod CPU/Memory charts tooltips

**Review by spadgett — APPROVED** [review]
/lgtm
/hold for approvals
/assign @yapei @reestr @ahardin-rh

File: frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx (hunk 1 above)
Comment (rebeccaalpert) [reply frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx:132]: I didn't seem to have an effect, but thanks for the the advice!

**Pull request merged**